core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -479,14 +480,14 @@ impl SchedulerController {
 
         const CHUNK_SIZE: usize = 128;
         let lock_results: [_; CHUNK_SIZE] = core::array::from_fn(|_| Ok(()));
```

Comment:
(off topic) this can be rewritten with inline const once we update to rust 1.79.